src/services/email.service.js

```diff
@@ -0,0 +1,40 @@
+const nodemailer = require('nodemailer');
+const { config } = require('dotenv');
+
+config();
+
+const transporter = nodemailer.createTransport({
+  host: process.env.SMTP_HOST,
+  port: process.env.SMTP_PORT,
+  auth: {
+    user: process.env.SMTP_USER,
+    pass: process.env.SMTP_PASSWORD,
+  },
+});
+
+function send({ email, subject, html }) {
+  return transporter.sendMail({
+    to: email,
+    subject,
+    html,
+  });
+}
+
+function sendActivationEmail(email, token) {
+  const href = `${process.env.CLIENT_HOST}/activate/${token}`;
```

Comment:
Ensure that `process.env.CLIENT_HOST` is correctly set in your environment to generate valid activation links.